x-pack/plugins/infra/types/eui_experimental.d.ts

```diff
@@ -7,11 +7,7 @@
 declare module '@elastic/eui/lib/experimental' {
   import { CommonProps } from '@elastic/eui/src/components/common';
   export type EuiSeriesChartProps = CommonProps & {
-    width?: number | string;
-    height?: number | string;
-    orientation?: string;
     xType?: string;
-    yType?: string;
```

Comment:
Looks like there's another addition to remove: ` | string` in  `line 24`: 
``` ts
data: Array<{ x: number; y: number | string; y0?: number }>;
```

![image](https://user-images.githubusercontent.com/2946766/57093611-94ac6e00-6ccb-11e9-897a-eff72058451a.png)

Reply:
🦅 👁 ! Thanks